skipruntime-ts/helpers/src/rest.ts

```diff
@@ -79,30 +104,55 @@ export class RESTWrapperOfSkipService {
     return data ?? [];
   }
 
+  /**
+   * Write the values for a single key in a collection.
+   * @param collection - name of the input collection to update, must be a key of the `Inputs` type parameter of the `SkipService` running at `entrypoint`
+   * @param key - key of entry to write
+   * @param values - values of entry to write
+   * @returns {void}
+   */
   async put<K extends Json, V extends Json>(
     collection: string,
     key: K,
-    value: V[],
+    values: V[],
   ): Promise<void> {
-    return await this.patch(collection, [[key, value]]);
+    return await this.patch(collection, [[key, values]]);
```

Comment:
The HTTP interface supports PUT, so I don't know why this is not implemented using it. Also, deleteKey could be implemented using PUT as well. Is there any reason to prefer the PATCH implementations? @beauby ?

Reply:
It might be to circumvent the key type issue. @skiplabsdaniel?